Cargo.toml

```diff
@@ -4,9 +4,4 @@
 h2 = { git = "https://github.com/kate-goldenring/h2", branch = "master" }
 
 [workspace]
```

Comment:
nit - formatting

Reply:
Maybe there is a way we can define/request this formatting to automatically be applied when `cargo fmt` is run by updating `rustfmt.toml`. I'll check it out

Reply:
rustfmt only works on `.rs` files. I tried to find a vscode extension that would do it, but no luck. Fixed in c711e9a.